<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="7" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13347" started-at="2020-06-15T22:53:01 ICT" finished-at="2020-06-15T22:53:15 ICT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13347" started-at="2020-06-15T22:53:01 ICT" finished-at="2020-06-15T22:53:15 ICT">
      <class name="poly.test.login">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:poly.test.login@6a79c292]" name="launchBrowser" is-config="true" duration-ms="5069" started-at="2020-06-15T22:53:01 ICT" finished-at="2020-06-15T22:53:06 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="LOG05()[pri:1, instance:poly.test.login@6a79c292]" name="LOG05" duration-ms="3824" started-at="2020-06-15T22:53:06 ICT" finished-at="2020-06-15T22:53:10 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG05 -->
        <test-method status="PASS" signature="LOG06()[pri:1, instance:poly.test.login@6a79c292]" name="LOG06" duration-ms="836" started-at="2020-06-15T22:53:10 ICT" finished-at="2020-06-15T22:53:11 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG06 -->
        <test-method status="PASS" signature="LOG07()[pri:1, instance:poly.test.login@6a79c292]" name="LOG07" duration-ms="734" started-at="2020-06-15T22:53:11 ICT" finished-at="2020-06-15T22:53:12 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG07 -->
        <test-method status="PASS" signature="LOG08()[pri:1, instance:poly.test.login@6a79c292]" name="LOG08" duration-ms="219" started-at="2020-06-15T22:53:12 ICT" finished-at="2020-06-15T22:53:12 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG08 -->
        <test-method status="PASS" signature="LOG09()[pri:1, instance:poly.test.login@6a79c292]" name="LOG09" duration-ms="1248" started-at="2020-06-15T22:53:12 ICT" finished-at="2020-06-15T22:53:13 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG09 -->
        <test-method status="FAIL" signature="LOG01()[pri:2, instance:poly.test.login@6a79c292]" name="LOG01" duration-ms="946" started-at="2020-06-15T22:53:13 ICT" finished-at="2020-06-15T22:53:14 ICT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [404 HTML Template by Colorlib] but found [HTTP Status 500 – Internal Server Error]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [404 HTML Template by Colorlib] but found [HTTP Status 500 – Internal Server Error]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at poly.test.login.LOG01(login.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG01 -->
        <test-method status="FAIL" signature="LOG02()[pri:3, instance:poly.test.login@6a79c292]" name="LOG02" duration-ms="86" started-at="2020-06-15T22:53:14 ICT" finished-at="2020-06-15T22:53:14 ICT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://localhost:8080/PS09298ASM/login.htm] but found [http://localhost:8080/PS09298ASM/index.jsp/manage/product.htm]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://localhost:8080/PS09298ASM/login.htm] but found [http://localhost:8080/PS09298ASM/index.jsp/manage/product.htm]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at poly.test.login.LOG02(login.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LOG02 -->
        <test-method status="PASS" signature="terminateBrowser()[pri:0, instance:poly.test.login@6a79c292]" name="terminateBrowser" is-config="true" duration-ms="313" started-at="2020-06-15T22:53:14 ICT" finished-at="2020-06-15T22:53:15 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminateBrowser -->
      </class> <!-- poly.test.login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
